generators/app/templates/scripts/_banner.js

```diff
@@ -0,0 +1,3 @@
+const pkg = require('../package.json');
+
+module.exports = `/*! @name ${pkg.name} @version ${pkg.version} @copyright ${pkg.author} @license ${pkg.license} */`
```

Comment:
Do we want this on multiple lines like before? seems like a waste of space to me, but I don't feel strongly that it has to be done on the same line.